src/pages/docs/chat/rooms/presence.mdx

```diff
@@ -67,18 +67,23 @@ val subscription = room.presence.subscribe { event: PresenceEvent ->
 </Code>
 
 <If lang="javascript">
-You can also subscribe to only specific presence events, or an array of presence events:
+You can filter for specific presence events using if/switch statements:
 
 <Code>
 ```javascript
-// Subscribe to only 'enter' events:
-const { unsubscribe } = room.presence.subscribe('enter', (event) => {
-  console.log(`${event.member.clientId} entered with data: ${event.member.data}`);
+// Filter for only 'enter' events:
+import { PresenceEventType } from '@ably/chat';
```

Comment:
I think we can collapse this into a single example and include the type in the message, like we have for Kotlin and Swift